engine/engine.go

```diff
@@ -518,10 +519,21 @@ func createEngine(terragruntOptions *options.TerragruntOptions) (*proto.EngineCl
 
 	terragruntOptions.Logger.Debugf("Creating engine %s", localEnginePath)
 
+	engineLogLevel := os.Getenv(EngineLogLevelEnv)
+	if engineLogLevel == "" {
+		engineLogLevel = terragruntOptions.LogLevel.String()
+		// turn off log formatting if disabled for terragrunt
+		if terragruntOptions.DisableLogFormatting {
+			engineLogLevel = hclog.Off.String()
```

Comment:
This is different. Disabling log formatting only makes the logs output in key-value pairs, it doesn't stop them from logging altogether.